meadow-client/src/MainFrame.purs

```diff
@@ -474,7 +475,7 @@ evalF (ApplyTransaction next) = do
    Nothing -> pure next
 
 evalF (NextBlock next) = do
-  modifying (_marloweState <<< _blockNum) (\x -> x + ((fromInt 1) :: BigInteger))
+  modifying (_marloweState <<< _blockNum) (\x -> x + ((fromIntegral 1) :: BlockNumber))
```

Comment:
You can replace this lambda with `add one`.

Reply:
even nicer